src/components/post-card/PostCard.js

```diff
@@ -111,33 +146,60 @@ export class PostCard extends PureComponent {
         </div>
 
         {
-          // todo: добавить label, в котором как children будет если showComments = true - надпись "Hide comments",
+          // rtodo: добавить label, в котором как children будет если
+          //  showComments = true - надпись "Hide comments",
           //    иначе - "Show comments"
           //    повесить на нее onClick событие this.onToggleComments
           //    как класс задать ей "btn btn-link"
+          <label className="btn btn-link" onClick={this.onToggleComments} >
+            { showComments ? "Hide comments" : "Show comments" }
+          </label>
         }
-        {/* todo создать div который будет как children содержать error, если !!error */}
 
-        {/* todo в строке ниже изменить условие если showComments = true то показываем <label>Comments:</label>*/}
-        {!!comments.length && <label>Comments:</label>}
+
+        {/* dtodo создать div который будет как children содержать error, если !!error */}
+
+    {!!error && <div> {error} </div>}
+
+        {/* fdtodo в строке ниже изменить условие если showComments
+        = true то показываем <label>Comments:</label>*/}
+
+        {showComments && <label>Comments:</label>}
         {
-          //todo если секция комментариев открыта, т.е. showComments = true
+          //аtodo если секция комментариев открыта, т.е. showComments = true
           //   и идет загрузка комментариев, т.е. isCommentsLoading = true
-          // показываем лоадинг индикатор (можно просто строку с надписью "Loading comments ..." в div)
+          // показываем лоадинг индикатор
+          // (можно просто строку с надписью "Loading comments ..." в div)
+          //   const {dot-3-move, dot-2-move,dot-1-move, rotate-move, {index} =
+          showComments && isCommentsLoading &&
+          <div className="container">
```

Comment:
Also, you can create a separate component for loader so it can be used in many other places